lib/nmatrix/io/csv_reader.rb

```diff
@@ -0,0 +1,153 @@
+#--
```

Comment:
Hi @v0dro , I still have some problem understanding this comment, I add this `csv_reader.rb` here intending that this is the csv reader to fall back on in case there's no `nmatrix-csv` gem. If I were right and this file doesn't exist, a user may not be capable of reading csv file with only nmatrix core installed, Strictly, this is not a part of the plugin (not added to gemspec).
So would you please provide some opinions about how to make this happen without this file ?
Will it just work by moving the logic in this file to `lib/nmatrix/csv.rb` (I think it will not work)?
Thanks!